react/components/atoms/switcher/switcher.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, select, text, number, boolean } from "@storybook/addon-knobs";
+
+import { Switcher } from "./switcher";
+
+storiesOf("Components/Atoms/Switcher", module)
+    .addDecorator(withKnobs)
+    .add("Switcher", () => {
+        const checked = boolean("checked", false);
+        const animationDuration = number("Animation Duration", 200);
+        const checkedText = text("Checked text", "Checked!");
+        const uncheckedText = text("Unchecked text", "Not checked!");

```

Comment:
Myabe make it more similar to the components-vue story, which does not have text defined.